lib/restaurant.rb

```diff
@@ -0,0 +1,35 @@
+class Restaurant
+  def view_menu
+    display.view_menu(menu)
+  end
+
+  def add(meal)
+    choice = meal.downcase.to_sym
```

Comment:
What happens to strings with spaces in them when converted .to_sym? Just wondering how this would impact your code if there was a menu item with two words. like a quattro formaggi (I have probably spelt this wrong)

Reply:
Yes you are right. it's a detail I purposely side stepped by having one word pizzas only :)

not too much code to make it multi-word compatible so probably should have implemented it